packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -486,7 +486,7 @@ export abstract class BasePHP implements IsomorphicLocalPHP {
 	/** @inheritDoc */
 	@rethrowFileSystemError('Could not move "{path}"')
 	mv(fromPath: string, toPath: string) {
-		this[__private__dont__use].FS.mv(fromPath, toPath);
+		this[__private__dont__use].FS.rename(fromPath, toPath);
```

Comment:
This should probably be done in a separate PR. I confirmed that the method `FS.rename` works as expected.

Reply:
Nice one! Yes, a separate PR would be nice to make browsing the change history easier.

Reply:
This is such a great find. Thank you so much @eliot-akira! This improves all Playground-based projects

Reply:
Pull request created:

- #445